meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -9,6 +9,8 @@ FILES_${PN} = " \
     ${bindir}/mender-monitord \
     ${sysconfdir}/mender-monitor \
     ${datadir}/mender-monitor \
+    /var/lib/mender-monitor \
```

Comment:
Should use `localstatedir` here as well.

Reply:
I thought about it, but got inspired by: [/var/lib/mender-configure](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-mender/mender-configure/mender-configure.inc#L15) ok I will fix. shall I also fix the configure recipe?

Reply:
Yes, please!

Reply:
https://github.com/mendersoftware/meta-mender/pull/1447